nodes/mysocketio/mysocketio.go

```diff
@@ -33,26 +36,41 @@ func (s *mySocketIO) PreDeploy(configName, labCADir, labCARoot string) error {
 	// utils.CreateDirectory(s.cfg.LabDir, 0777)
 	return nil
 }
+func (s *mySocketIO) Deploy(ctx context.Context) error { return nil }
```

Comment:
why the creation of container was removed here?

Reply:
oops. fixed